specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2270 @@
+            "required": false,
+            "type": "string",
+            "pattern": "^\d+(-\d+)?$"
+          },	  
+          {
+            "$ref": "#/parameters/FileStream"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Request is queued successfully.",
+            "headers": {
+              "Operation-Location": {
+                "type": "string",
+                "description": "URL containing the resultId used to track the progress and obtain the result of the analyze operation."
+              }
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Analyze Business Card": {
+            "$ref": "./examples/BusinessCardBatch.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/prebuilt/businessCard/analyzeResults/{resultId}": {
+      "get": {
+        "summary": "Get Analyze Business Card Result",
+        "description": "Track the progress and obtain the result of the analyze business card operation.",
+        "operationId": "GetAnalyzeBusinessCardResult",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "resultId",
+            "in": "path",
+            "description": "Analyze operation result identifier.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeOperationResult"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get analyze business card result": {
+            "$ref": "./examples/BusinessCardBatchResult.json"
+          }
+        }
+      }
+    },
+    "/prebuilt/invoice/analyze": {
+      "post": {
+        "summary": "Analyze Invoice Document",
+        "description": "Extract field text and semantic values from a given invoice document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png', 'image/tiff' or "image/bmp". Alternatively, use 'application/json' type to specify the location (Uri) of the document to be analyzed.",
+        "operationId": "AnalyzeInvoiceAsync",
+        "consumes": [
+        ],
```

Comment:
needs "consumes" defined. edit: seems to be the case for many of the endpoints. are you getting some error when you add it?

Reply:
done.